src/printer.js

```diff
@@ -244,8 +244,25 @@ function printExpression(path, options, print) {
           ),
           getEncapsedQuotes(node, { opening: false })
         ]);
-      case "inline":
-        return node.value;
+      case "inline": {
+        const parentNode = path.getParentNode();
+        const isFirstNode =
```

Comment:
maybe helpful to add these `isFirstNode(path)` and `isLastNode(path)` to that util file you added?